### PR TITLE
Drop usage date and only update consent once

### PR DESCRIPTION
**Pull request by arothuis**
[130002585](https://www.pivotaltracker.com/story/show/130002585)
OpenConext/OpenConext-engineblock#316

This PR prevents consent_date from updating upon each consent and removes the tracking of the last date on which consent has been given. The usage_date column has been removed from the consent table.

This impacts Profile, as a consumer of the Consent API, see: OpenConext/OpenConext-profile#73

### CODE REVIEW

**Comment by arothuis** [comment]
Will wait before merging until OpenConext/OpenConext-profile#73 lands.

File: src/OpenConext/EngineBlock/Authentication/Dto/Consent.php [hunk 1]
```diff
@@ -67,7 +67,6 @@ function (ContactPerson $contact) {
         return [
             'service_provider' => $serviceProvider,
             'consent_given_on' => $this->consent->getDateConsentWasGivenOn()->format(DateTime::ATOM),
-            'last_used_on'     => $this->consent->getDateLastUsedOn()->format(DateTime::ATOM),
```
Comment (arothuis) [inline src/OpenConext/EngineBlock/Authentication/Dto/Consent.php]: This is indeed a BC break, but it is reasonable to assume Profile is the only consumer for this client.

Do you suggest adding an update note somewhere?

Comment (arothuis) [inline src/OpenConext/EngineBlock/Authentication/Dto/Consent.php]: Discussed it with SURF, there are no other consumers.
I updated the docs for migrating to 5.2, mentioning the changes.

Comment (rjkip) [inline src/OpenConext/EngineBlock/Authentication/Dto/Consent.php]: :+1:

**Pull request merged**